peer.go

```diff
@@ -111,13 +111,15 @@ type peer struct {
 	addr        *lnwire.NetAddress
 	pubKeyBytes [33]byte
 
+	// startTime is the time this peer connection was successfully
+	// established. It will be zero for peers that did not successfuly
+	// Start().
+	startTime time.Time
+
 	inbound bool
 
 	// This mutex protects all the stats below it.
 	sync.RWMutex
-	timeConnected time.Time
```

Comment:
The mutex guarding these is also not being used at the moment so it can be removed.